ui/src/testing/stubs/notebooks-api-stub.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from 'app/pages/analysis/util';
 
 export class NotebooksApiStub extends NotebooksApi {
-  notebookList: FileDetail[];
+  public notebookList: FileDetail[];
```

Comment:
Made this public so that you can modify attributes for individual test. For instance, I needed to adjust sizeInBytes in a couple tests to ensure that a modal was opening correctly.